src/genie/libs/parser/iosxe/tests/test_show_cts.py

```diff
@@ -51,6 +52,153 @@ def test_show_cts_pacs_empty(self):
         with self.assertRaises(SchemaEmptyParserError):
             parsed_output = obj.parse()
 
+# =============================
+# Unit test for 'show cts role-based permissions'
+# =============================
+class TestShowCtsRoleBasedPermissions(unittest.TestCase):
+    """Unit test for 'show cts role-based permissions'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+        "rbp_policies": {
+            1: {
```

Comment:
I believe this number is index. if so, better to have key name as `indexes` like below:
```
"indexes": {
    1: {
        "policy_name": "default",
        (snip)

Reply:
added the same to https://github.com/CiscoTestAutomation/genieparser/pull/250